modules/terminate-agent-hook/iam.tf

```diff
@@ -87,14 +103,56 @@ data "aws_iam_policy_document" "lambda" {
   }
 }
 
+data "aws_iam_policy_document" "ssh_key_housekeeping" {
+  statement {
+    sid = "SSHKeyHousekeepingList"
+
+    effect = "Allow"
+    actions = [
+      "ec2:DescribeKeyPairs"
```

Comment:
Not having much time, but seems DescribeKeyPairs is part of 2 policies, is this correct?

Reply:
removed